docker-compose.yml

```diff
@@ -64,6 +64,7 @@ x-services:
       - --developer
       - --dev-fast-gossip
       - --dev-fast-reconnect
+      - --experimental-offers
```

Comment:
Absolutely no reason for which we should enable them, because neither frontend nor backend support them